app/views/publishers/vacancies/extend_deadline/show.html.slim

```diff
@@ -24,7 +24,7 @@
           = f.govuk_date_field :starts_on
 
           .clear-form__checkbox
-            = f.govuk_check_box :starts_asap, true, 0, multiple: false, link_errors: false
+            = f.govuk_check_box :starts_asap, "true", 0, multiple: false, link_errors: false
```

Comment:
true => "true" seems odd, sure you have your reasons?

Reply:
If you go to that page now, check starts_asap but don't enter an expiry date (so validation fails) and submit the form, the starts_asap won't be checked, but this fixes that